### ISSUE TITLE
something about feature extraction

### ISSUE THREAD

**Issue by BinchaoPeng** [jerryji1993/DNABERT#20]
Hello，I seemly do not find how to get feature extraction vector with your pre-trained model in your readme.md.  So I use your dnaber in pytorch-transformer(Huggingface/transformer is from pytorch-transformer),but i always got a vector of [batchsize,1,768], not the right dim [batchsize,Kmer-len,768].

And, I want to konw how I can use dnabert to obtain feature extraction vector, or do you support the function of feature extraction?

sincerely,PBC

**Comment by Zhihan1996** [comment]
Hi,

Please refer to https://github.com/jerryji1993/DNABERT/issues/11 and https://github.com/jerryji1993/DNABERT/issues/16 for the feature extraction.

**Comment by jerryji1993** [comment]
Closed #20.
